src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListenerOptions.cs

```diff
@@ -25,36 +25,30 @@ namespace OpenTelemetry.Exporter.Prometheus
     /// </summary>
     public class PrometheusHttpListenerOptions
     {
-        private IReadOnlyCollection<string> prefixes = new string[] { "http://localhost:9464/" };
+        private IReadOnlyCollection<string> uriPrefixes = new string[] { "http://localhost:9464/" };
 
         /// <summary>
         /// Gets or sets the path to use for the scraping endpoint. Default value: "/metrics".
         /// </summary>
         public string ScrapeEndpointPath { get; set; } = "/metrics";
 
         /// <summary>
-        /// Gets or sets the prefixes to use for the http listener.
+        /// Gets or sets the URI (Uniform Resource Identifier) prefixes to use for the http listener.
         /// Default value: <c>["http://localhost:9464/"]</c>.
         /// </summary>
-        public IReadOnlyCollection<string> Prefixes
+        public IReadOnlyCollection<string> UriPrefixes
         {
-            get => this.prefixes;
+            get => this.uriPrefixes;
             set
             {
                 Guard.ThrowIfNull(value);
 
-                foreach (string inputUri in value)
+                if (value.Count == 0)
```

Comment:
This might be a good candidate for a new Guard method. `Guard.ThrowIfCollectionEmpty`

Reply:
Do you see many other places having the same need, so we need abstraction?

Reply:
No, this is the first place I've noticed.

Reply:
I'll leave it as-is for now.